java/src/jmri/jmrit/logix/Engineer.java

```diff
@@ -64,7 +64,7 @@
 
     @Override
     public void run() {
-        log.debug("Engineer started warrant {}", _warrant.getDisplayName());
+        if (debug) log.debug("Engineer started warrant {}", _warrant.getDisplayName());
```

Comment:
There's a number of places where this is happening, so that's going to be addressed in a separate PR.